src/main/java/com/kodilla/ecommercee/domain/CartDto.java

```diff
@@ -1,10 +1,12 @@
 package com.kodilla.ecommercee.domain;
 
+import lombok.AllArgsConstructor;
 import lombok.Getter;
 import lombok.Setter;
 
 @Getter
 @Setter
+@AllArgsConstructor
```

Comment:
Z tego co pamietam w zadaniu musi byc możliwość utworzenia pustego koszyka, Wiec jeśli będzie @AllArgsConstructor to juz pustego nie zrobisz